src/components/FeedPortal/index.jsx

```diff
@@ -1,24 +1,28 @@
-import React, { Fragment } from "react"
+import React, { Component, Fragment } from "react"
 import "./FeedPortal.css"
-import teamQuery from './graphql/teamQuery';
-import Request from "../utilities/Request"
 import SideBar from './components/SideBar';
 import NewsFeed from './components/NewsFeed';
 
-const FeedPortal = ({ data: { user } }) => {
-  return (
-    <div className="view-container">
-      <div className="portal-container">
-        <SideBar />
-        <NewsFeed />
+class FeedPortal extends Component {
+  state = {
+    newsfeed: {
+      type: "ALL",
+      team_id: null
+    }
+  }
+
+  toggleNewsFeed = (type, team_id) => this.setState({ type, team_id })
```

Comment:
should this be
`this.setState({ newsfeed: {type, team_id} })` ?

Reply:
Yes, good catch